emails/migrations/0046_profile_remove_level_one_email_trackers.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-06-14 05:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("emails", "0045_deletedaddress_num_replied"),
```

Comment:
praise: mostly for Django compared to node/express/knex - yay migrations are separated by apps so I don't have to worry about this conflicting with my phone migrations!